docs/sphinx/user/build.rst

```diff
@@ -126,6 +126,11 @@ Other AMR-Wind specific options
 
    Directory for saving gold files when using :program:`fcompare`. Default: ""
 
+.. cmakeval:: AMR_WIND_TEST_WITH_PYTHON
+
+   Enable performing tests that rely on python, primarily consists of checks 
+   for post-processing scripts in tools directory. Default: OFF
+
```

Comment:
I said "primarily" here because I'm assuming this option is also needed for the verification test (not currently running) that has a python script. This PR doesn't hook up this option to the verification test, but that might need to be done in the future